test/src/module/command/restoreTest.c

```diff
@@ -2197,6 +2199,51 @@ testRun(void)
         TEST_RESULT_LOG("P00   INFO: write updated " TEST_PATH "/pg/postgresql.auto.conf");
     }
 
+    // *****************************************************************************************************************************
+    if (testBegin("restoreJobResult()"))
+    {
+        // Set log level to detail
+        harnessLogLevelSet(logLevelDetail);
+
+        // -------------------------------------------------------------------------------------------------------------------------
+        TEST_TITLE("report host/100% progress on noop result");
+
+        // Create job that skips file
+        ProtocolParallelJob *job = protocolParallelJobNew(VARSTRDEF("pg_data/test"), protocolCommandNew(strIdFromZ("x")));
+
+        PackWrite *const resultPack = protocolPackNew();
+        pckWriteStrP(resultPack, STRDEF("pg_data/test"));
+        pckWriteU32P(resultPack, backupCopyResultNoOp);
```

Comment:
I think using `restoreResultZero` is more appropriate in this context.
```suggestion
        pckWriteU32P(resultPack, restoreResultZero);
```

Reply:
Also `#include "command/backup/file.h` is no longer needed in `restoreTest.c`

Reply:
Good catch, thanks!